metaci/build/models.py

```diff
@@ -660,6 +660,21 @@ def run_flow(self, project_config, org_config):
 
         flow_config = project_config.get_flow(self.flow)
 
+        if (
+            self.build.plan.role == "release"
+            and self.build.release
+            and flow_config.name == "release_production"
+        ):
+            for k, v in flow_config.config["steps"].items():
+                if v["task"] == "github_release_notes":
+                    v["options"] = {
+                        "tag": "^^github_release.tag_name",
+                        "publish": True,
+                        "version_id": "^^upload_production.version_id",
+                        "sandbox_date": self.build.release.sandbox_push_date,
+                        "production_date": self.build.release.production_push_date,
+                    }
```

Comment:
It should not be necessary to do it this way. The FlowCoordinator is supposed to support overriding task options without needing to change the flow config. Also we should only adjust the options that come from the Release object; the others should still come from cumulusci.yml.

I'll take a look at why passing options to the FlowCoordinator wasn't working when I get a chance.

Reply:
Yeah I had to step deep into Pdb to get this to work. Appreciate it @davisagli